src/osd/OSD.cc

```diff
@@ -8262,6 +8262,19 @@ void OSD::handle_osd_map(MOSDMap *m)
 	  << dendl;
 
   logger->inc(l_osd_map);
+  // since the map received counter is being incremented we know that the osd has recieved a map
+  // first check if it contains a full map in which case increase that logger count by 1 
+  if(!m->maps.empty()){
+     logger ->inc(l_osd_map_full,m->maps.size());
```

Comment:
No space between ```logger``` and ```->```.  There *should* be a space between ```,``` and ```m->maps.size():

```logger->inc(l_osd_map_full, m->maps.size()```